From f8c2f5f516088e6feb1a06d2dbf575c4c78cba96 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Wed, 31 Jan 2018 10:57:35 +0800
Subject: [PATCH 3293/5242] MLK-17455: ARM64: dts: correct the pinctrl setting
 for audio peripheral

commit  3943db7aa86fadbccd3608bf4838a45e5b5dfa76 from
https://source.codeaurora.org/external/imx/linux-imx.git

According to the Reference manual, the bit 1-4 of PAD setting is reserved.

Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../dts/freescale/fsl-imx8qm-lpddr4-arm2-mqs.dts   |   24 ++++++++--------
 .../dts/freescale/fsl-imx8qm-lpddr4-arm2-spdif.dts |   24 ++++++++--------
 .../boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts  |   22 +++++++-------
 arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts   |   30 ++++++++++----------
 .../freescale/fsl-imx8qxp-lpddr4-arm2-spdif.dts    |    4 +--
 .../freescale/fsl-imx8qxp-lpddr4-arm2-wm8962.dts   |   16 +++++------
 .../boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts |   22 +++++++-------
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts  |   30 ++++++++++----------
 8 files changed, 86 insertions(+), 86 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-mqs.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-mqs.dts
index 9b5a598..64d0bb2 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-mqs.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-mqs.dts
@@ -110,8 +110,8 @@
 
 		pinctrl_spdif0: spdif0grp {
 			fsl,pins = <
-				SC_P_SPDIF0_TX_AUD_SPDIF0_TX	0xc600004c
-				SC_P_SPDIF0_RX_AUD_SPDIF0_RX	0xc600004c
+				SC_P_SPDIF0_TX_AUD_SPDIF0_TX	0xc6000040
+				SC_P_SPDIF0_RX_AUD_SPDIF0_RX	0xc6000040
 			>;
 		};
 
@@ -124,16 +124,16 @@
 
 		pinctrl_sai0: sai0grp {
 			fsl,pins = <
-				SC_P_SAI1_RXC_AUD_SAI0_TXD		0xc600006c
-				SC_P_SAI1_RXFS_AUD_SAI0_RXD		0xc600004c
-				SC_P_SAI1_TXC_AUD_SAI0_TXC		0xc600004c
-				SC_P_SPI2_CS1_AUD_SAI0_TXFS		0xc600004c
-
-				SC_P_USDHC2_CD_B_LSIO_GPIO4_IO12	0xc600004c
-				SC_P_USDHC2_DATA0_LSIO_GPIO5_IO26	0xc600004c
-				SC_P_USDHC2_DATA1_LSIO_GPIO5_IO27	0xc600004c
-				SC_P_LVDS1_I2C0_SDA_LSIO_GPIO1_IO13	0xc600004c
-				SC_P_MCLK_OUT0_AUD_ACM_MCLK_OUT0	0xc600004c
+				SC_P_SAI1_RXC_AUD_SAI0_TXD		0xc6000060
+				SC_P_SAI1_RXFS_AUD_SAI0_RXD		0xc6000040
+				SC_P_SAI1_TXC_AUD_SAI0_TXC		0xc6000040
+				SC_P_SPI2_CS1_AUD_SAI0_TXFS		0xc6000040
+
+				SC_P_USDHC2_CD_B_LSIO_GPIO4_IO12	0xc6000040
+				SC_P_USDHC2_DATA0_LSIO_GPIO5_IO26	0xc6000040
+				SC_P_USDHC2_DATA1_LSIO_GPIO5_IO27	0xc6000040
+				SC_P_LVDS1_I2C0_SDA_LSIO_GPIO1_IO13	0xc6000040
+				SC_P_MCLK_OUT0_AUD_ACM_MCLK_OUT0	0xc6000040
 			>;
 		};
 	};
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-spdif.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-spdif.dts
index db8a047..8125de8 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-spdif.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-spdif.dts
@@ -109,8 +109,8 @@
 
 		pinctrl_spdif0: spdif0grp {
 			fsl,pins = <
-				SC_P_SPDIF0_TX_AUD_SPDIF0_TX	0xc600004c
-				SC_P_SPDIF0_RX_AUD_SPDIF0_RX	0xc600004c
+				SC_P_SPDIF0_TX_AUD_SPDIF0_TX	0xc6000040
+				SC_P_SPDIF0_RX_AUD_SPDIF0_RX	0xc6000040
 			>;
 		};
 
@@ -123,16 +123,16 @@
 
 		pinctrl_sai0: sai0grp {
 			fsl,pins = <
-				SC_P_SAI1_RXC_AUD_SAI0_TXD		0xc600006c
-				SC_P_SAI1_RXFS_AUD_SAI0_RXD		0xc600004c
-				SC_P_SAI1_TXC_AUD_SAI0_TXC		0xc600004c
-				SC_P_SPI2_CS1_AUD_SAI0_TXFS		0xc600004c
-
-				SC_P_USDHC2_CD_B_LSIO_GPIO4_IO12	0xc600004c
-				SC_P_USDHC2_DATA0_LSIO_GPIO5_IO26	0xc600004c
-				SC_P_USDHC2_DATA1_LSIO_GPIO5_IO27	0xc600004c
-				SC_P_LVDS1_I2C0_SDA_LSIO_GPIO1_IO13	0xc600004c
-				SC_P_MCLK_OUT0_AUD_ACM_MCLK_OUT0	0xc600004c
+				SC_P_SAI1_RXC_AUD_SAI0_TXD		0xc6000060
+				SC_P_SAI1_RXFS_AUD_SAI0_RXD		0xc6000040
+				SC_P_SAI1_TXC_AUD_SAI0_TXC		0xc6000040
+				SC_P_SPI2_CS1_AUD_SAI0_TXFS		0xc6000040
+
+				SC_P_USDHC2_CD_B_LSIO_GPIO4_IO12	0xc6000040
+				SC_P_USDHC2_DATA0_LSIO_GPIO5_IO26	0xc6000040
+				SC_P_USDHC2_DATA1_LSIO_GPIO5_IO27	0xc6000040
+				SC_P_LVDS1_I2C0_SDA_LSIO_GPIO1_IO13	0xc6000040
+				SC_P_MCLK_OUT0_AUD_ACM_MCLK_OUT0	0xc6000040
 			>;
 		};
 	};
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
index 1d8751f..13dd2dbc 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
@@ -206,17 +206,17 @@
 
 		pinctrl_esai0: esai0grp {
 			fsl,pins = <
-				SC_P_ESAI0_FSR_AUD_ESAI0_FSR		0xc600004c
-				SC_P_ESAI0_FST_AUD_ESAI0_FST		0xc600004c
-				SC_P_ESAI0_SCKR_AUD_ESAI0_SCKR		0xc600004c
-				SC_P_ESAI0_SCKT_AUD_ESAI0_SCKT		0xc600004c
-				SC_P_ESAI0_TX0_AUD_ESAI0_TX0		0xc600004c
-				SC_P_ESAI0_TX1_AUD_ESAI0_TX1		0xc600004c
-				SC_P_ESAI0_TX2_RX3_AUD_ESAI0_TX2_RX3	0xc600004c
-				SC_P_ESAI0_TX3_RX2_AUD_ESAI0_TX3_RX2	0xc600004c
-				SC_P_ESAI0_TX4_RX1_AUD_ESAI0_TX4_RX1	0xc600004c
-				SC_P_ESAI0_TX5_RX0_AUD_ESAI0_TX5_RX0	0xc600004c
-				SC_P_MCLK_OUT0_AUD_ACM_MCLK_OUT0	0xc600004c
+				SC_P_ESAI0_FSR_AUD_ESAI0_FSR		0xc6000040
+				SC_P_ESAI0_FST_AUD_ESAI0_FST		0xc6000040
+				SC_P_ESAI0_SCKR_AUD_ESAI0_SCKR		0xc6000040
+				SC_P_ESAI0_SCKT_AUD_ESAI0_SCKT		0xc6000040
+				SC_P_ESAI0_TX0_AUD_ESAI0_TX0		0xc6000040
+				SC_P_ESAI0_TX1_AUD_ESAI0_TX1		0xc6000040
+				SC_P_ESAI0_TX2_RX3_AUD_ESAI0_TX2_RX3	0xc6000040
+				SC_P_ESAI0_TX3_RX2_AUD_ESAI0_TX3_RX2	0xc6000040
+				SC_P_ESAI0_TX4_RX1_AUD_ESAI0_TX4_RX1	0xc6000040
+				SC_P_ESAI0_TX5_RX0_AUD_ESAI0_TX5_RX0	0xc6000040
+				SC_P_MCLK_OUT0_AUD_ACM_MCLK_OUT0	0xc6000040
 			>;
 		};
 
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
index fb2fa75..272259c 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
@@ -218,16 +218,16 @@
 
 		pinctrl_esai0: esai0grp {
 			fsl,pins = <
-				SC_P_ESAI0_FSR_AUD_ESAI0_FSR            0xc600004c
-				SC_P_ESAI0_FST_AUD_ESAI0_FST            0xc600004c
-				SC_P_ESAI0_SCKR_AUD_ESAI0_SCKR          0xc600004c
-				SC_P_ESAI0_SCKT_AUD_ESAI0_SCKT          0xc600004c
-				SC_P_ESAI0_TX0_AUD_ESAI0_TX0            0xc600004c
-				SC_P_ESAI0_TX1_AUD_ESAI0_TX1            0xc600004c
-				SC_P_ESAI0_TX2_RX3_AUD_ESAI0_TX2_RX3    0xc600004c
-				SC_P_ESAI0_TX3_RX2_AUD_ESAI0_TX3_RX2    0xc600004c
-				SC_P_ESAI0_TX4_RX1_AUD_ESAI0_TX4_RX1    0xc600004c
-				SC_P_ESAI0_TX5_RX0_AUD_ESAI0_TX5_RX0    0xc600004c
+				SC_P_ESAI0_FSR_AUD_ESAI0_FSR            0xc6000040
+				SC_P_ESAI0_FST_AUD_ESAI0_FST            0xc6000040
+				SC_P_ESAI0_SCKR_AUD_ESAI0_SCKR          0xc6000040
+				SC_P_ESAI0_SCKT_AUD_ESAI0_SCKT          0xc6000040
+				SC_P_ESAI0_TX0_AUD_ESAI0_TX0            0xc6000040
+				SC_P_ESAI0_TX1_AUD_ESAI0_TX1            0xc6000040
+				SC_P_ESAI0_TX2_RX3_AUD_ESAI0_TX2_RX3    0xc6000040
+				SC_P_ESAI0_TX3_RX2_AUD_ESAI0_TX3_RX2    0xc6000040
+				SC_P_ESAI0_TX4_RX1_AUD_ESAI0_TX4_RX1    0xc6000040
+				SC_P_ESAI0_TX5_RX0_AUD_ESAI0_TX5_RX0    0xc6000040
 			>;
 		};
 
@@ -330,11 +330,11 @@
 
 		pinctrl_sai1: sai1grp {
 			fsl,pins = <
-				SC_P_SAI1_RXD_AUD_SAI1_RXD		0x0600004c
-				SC_P_SAI1_RXC_AUD_SAI1_RXC		0x0600004c
-				SC_P_SAI1_RXFS_AUD_SAI1_RXFS		0x0600004c
-				SC_P_SAI1_TXD_AUD_SAI1_TXD		0x0600006c
-				SC_P_SAI1_TXC_AUD_SAI1_TXC		0x0600004c
+				SC_P_SAI1_RXD_AUD_SAI1_RXD		0x06000040
+				SC_P_SAI1_RXC_AUD_SAI1_RXC		0x06000040
+				SC_P_SAI1_RXFS_AUD_SAI1_RXFS		0x06000040
+				SC_P_SAI1_TXD_AUD_SAI1_TXD		0x06000060
+				SC_P_SAI1_TXC_AUD_SAI1_TXC		0x06000040
 			>;
 		};
 
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-spdif.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-spdif.dts
index 4d1688d..f333c2b 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-spdif.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-spdif.dts
@@ -35,8 +35,8 @@
 
 		pinctrl_spdif0: spdif0grp {
 			fsl,pins = <
-				SC_P_SPDIF0_TX_ADMA_SPDIF0_TX	0xc600004c
-				SC_P_SPDIF0_RX_ADMA_SPDIF0_RX	0xc600004c
+				SC_P_SPDIF0_TX_ADMA_SPDIF0_TX	0xc6000040
+				SC_P_SPDIF0_RX_ADMA_SPDIF0_RX	0xc6000040
 			>;
 		};
 
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-wm8962.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-wm8962.dts
index 9a65568..8f39af8 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-wm8962.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-wm8962.dts
@@ -62,14 +62,14 @@
 	imx8qxp-lpddr4-arm2 {
 		pinctrl_sai0: sai0grp {
 			fsl,pins = <
-				SC_P_SAI0_TXFS_ADMA_SAI0_TXFS		0xc600004c
-				SC_P_SAI0_TXC_ADMA_SAI0_TXC		0xc600004c
-				SC_P_SAI0_TXD_ADMA_SAI0_TXD		0xc600006c
-				SC_P_SAI0_RXD_ADMA_SAI0_RXD		0xc600004c
-				SC_P_SPI2_SDO_LSIO_GPIO1_IO01		0xc600004c
-				SC_P_SPI2_SDI_LSIO_GPIO1_IO02		0xc600004c
-				SC_P_SPI2_SCK_LSIO_GPIO1_IO03		0xc600004c
-				SC_P_MCLK_OUT0_ADMA_ACM_MCLK_OUT0	0xc600004c
+				SC_P_SAI0_TXFS_ADMA_SAI0_TXFS		0xc6000040
+				SC_P_SAI0_TXC_ADMA_SAI0_TXC		0xc6000040
+				SC_P_SAI0_TXD_ADMA_SAI0_TXD		0xc6000060
+				SC_P_SAI0_RXD_ADMA_SAI0_RXD		0xc6000040
+				SC_P_SPI2_SDO_LSIO_GPIO1_IO01		0xc6000040
+				SC_P_SPI2_SDI_LSIO_GPIO1_IO02		0xc6000040
+				SC_P_SPI2_SCK_LSIO_GPIO1_IO03		0xc6000040
+				SC_P_MCLK_OUT0_ADMA_ACM_MCLK_OUT0	0xc6000040
 			>;
 		};
 	};
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
index 2a7702a..9ff6f27 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
@@ -166,17 +166,17 @@
 
 		pinctrl_esai0: esai0grp {
 			fsl,pins = <
-				SC_P_ESAI0_FSR_ADMA_ESAI0_FSR		0xc600004c
-				SC_P_ESAI0_FST_ADMA_ESAI0_FST		0xc600004c
-				SC_P_ESAI0_SCKR_ADMA_ESAI0_SCKR		0xc600004c
-				SC_P_ESAI0_SCKT_ADMA_ESAI0_SCKT		0xc600004c
-				SC_P_ESAI0_TX0_ADMA_ESAI0_TX0		0xc600004c
-				SC_P_ESAI0_TX1_ADMA_ESAI0_TX1		0xc600004c
-				SC_P_ESAI0_TX2_RX3_ADMA_ESAI0_TX2_RX3	0xc600004c
-				SC_P_ESAI0_TX3_RX2_ADMA_ESAI0_TX3_RX2	0xc600004c
-				SC_P_ESAI0_TX4_RX1_ADMA_ESAI0_TX4_RX1	0xc600004c
-				SC_P_ESAI0_TX5_RX0_ADMA_ESAI0_TX5_RX0	0xc600004c
-				SC_P_MCLK_OUT0_ADMA_ACM_MCLK_OUT0	0xc600004c
+				SC_P_ESAI0_FSR_ADMA_ESAI0_FSR		0xc6000040
+				SC_P_ESAI0_FST_ADMA_ESAI0_FST		0xc6000040
+				SC_P_ESAI0_SCKR_ADMA_ESAI0_SCKR		0xc6000040
+				SC_P_ESAI0_SCKT_ADMA_ESAI0_SCKT		0xc6000040
+				SC_P_ESAI0_TX0_ADMA_ESAI0_TX0		0xc6000040
+				SC_P_ESAI0_TX1_ADMA_ESAI0_TX1		0xc6000040
+				SC_P_ESAI0_TX2_RX3_ADMA_ESAI0_TX2_RX3	0xc6000040
+				SC_P_ESAI0_TX3_RX2_ADMA_ESAI0_TX3_RX2	0xc6000040
+				SC_P_ESAI0_TX4_RX1_ADMA_ESAI0_TX4_RX1	0xc6000040
+				SC_P_ESAI0_TX5_RX0_ADMA_ESAI0_TX5_RX0	0xc6000040
+				SC_P_MCLK_OUT0_ADMA_ACM_MCLK_OUT0	0xc6000040
 			>;
 		};
 
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
index 4393a42..aef3f0d 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
@@ -213,16 +213,16 @@
 
 		pinctrl_esai0: esai0grp {
 			fsl,pins = <
-				SC_P_ESAI0_FSR_ADMA_ESAI0_FSR		0xc600004c
-				SC_P_ESAI0_FST_ADMA_ESAI0_FST		0xc600004c
-				SC_P_ESAI0_SCKR_ADMA_ESAI0_SCKR		0xc600004c
-				SC_P_ESAI0_SCKT_ADMA_ESAI0_SCKT		0xc600004c
-				SC_P_ESAI0_TX0_ADMA_ESAI0_TX0		0xc600004c
-				SC_P_ESAI0_TX1_ADMA_ESAI0_TX1		0xc600004c
-				SC_P_ESAI0_TX2_RX3_ADMA_ESAI0_TX2_RX3	0xc600004c
-				SC_P_ESAI0_TX3_RX2_ADMA_ESAI0_TX3_RX2	0xc600004c
-				SC_P_ESAI0_TX4_RX1_ADMA_ESAI0_TX4_RX1	0xc600004c
-				SC_P_ESAI0_TX5_RX0_ADMA_ESAI0_TX5_RX0	0xc600004c
+				SC_P_ESAI0_FSR_ADMA_ESAI0_FSR		0xc6000040
+				SC_P_ESAI0_FST_ADMA_ESAI0_FST		0xc6000040
+				SC_P_ESAI0_SCKR_ADMA_ESAI0_SCKR		0xc6000040
+				SC_P_ESAI0_SCKT_ADMA_ESAI0_SCKT		0xc6000040
+				SC_P_ESAI0_TX0_ADMA_ESAI0_TX0		0xc6000040
+				SC_P_ESAI0_TX1_ADMA_ESAI0_TX1		0xc6000040
+				SC_P_ESAI0_TX2_RX3_ADMA_ESAI0_TX2_RX3	0xc6000040
+				SC_P_ESAI0_TX3_RX2_ADMA_ESAI0_TX3_RX2	0xc6000040
+				SC_P_ESAI0_TX4_RX1_ADMA_ESAI0_TX4_RX1	0xc6000040
+				SC_P_ESAI0_TX5_RX0_ADMA_ESAI0_TX5_RX0	0xc6000040
 			>;
 		};
 
@@ -334,11 +334,11 @@
 
 		pinctrl_sai1: sai1grp {
 			fsl,pins = <
-				SC_P_SAI1_RXD_ADMA_SAI1_RXD	0x0600004c
-				SC_P_SAI1_RXC_ADMA_SAI1_TXC	0x0600004c
-				SC_P_SAI1_RXFS_ADMA_SAI1_TXFS	0x0600004c
-				SC_P_SPI0_CS1_ADMA_SAI1_TXD	0x0600006c
-				SC_P_SPI2_CS0_LSIO_GPIO1_IO00	0x0600004c
+				SC_P_SAI1_RXD_ADMA_SAI1_RXD	0x06000040
+				SC_P_SAI1_RXC_ADMA_SAI1_TXC	0x06000040
+				SC_P_SAI1_RXFS_ADMA_SAI1_TXFS	0x06000040
+				SC_P_SPI0_CS1_ADMA_SAI1_TXD	0x06000060
+				SC_P_SPI2_CS0_LSIO_GPIO1_IO00	0x06000040
 			>;
 		};
 
-- 
1.7.9.5

